patterns/2-structured/project-setup/templates/CODE_OF_CONDUCT-template.md

```diff
@@ -0,0 +1,78 @@
+<!--
+Variables to be replaced:
+- {{ Community-Name }}
+- {{ CommunityMail-Link }}
+-->
+
+# Code of Conduct
```

Comment:
You mention that this template is adapted from https://www.contributor-covenant.org/version/2/1/code_of_conduct/.

Could you add some context to the major changes you made, so that I can understand why they were made?

The changes I spotted:

- left away the 2nd sentence of pledge 
> "We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community."
- renamed "Enforcement Responsibilities" to "Our Responsibilities"
- left away the "Enforcement Guidelines"

I am sure I missed other changes that were made :)